frontend/app/assets/javascripts/spree/frontend/views/spree/products/cart_form.js

```diff
@@ -263,6 +263,7 @@ Spree.ready(function($) {
           ), Spree.variantById($cartForm, variantId))
         },
         function(_error) {
+          if (_error !== '') document.querySelector('#no-product-available .no-product-available-text').innerText = _error
```

Comment:
Can we add brackets here?

```
if (error !== '') {
  ...
}
```

Reply:
And also let's check if `error` has type `String`